phpunit.xml.dist

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit bootstrap="tests/_bootstrap.php" colors="true">
+	<php>
+		<server name="KERNEL_DIR" value="app/" />
+	</php>
+	<testsuites>
+		<testsuite>
+			<directory suffix="Test.php">./tests/functional</directory>
+			<directory suffix="Test.php">./tests/unit</directory>
+        </testsuite>
+	</testsuites>
+	<filter>
+		<whitelist>
+			<directory>./libraries/incubator</directory>
+		</whitelist>
+	</filter>
+	<logging>
+		<log type="coverage-clover" target="build/logs/clover.xml" />
+		<log type="coverage-html" target="build/reports" lowUpperBound="35" highLowerBound="70" />
+	</logging>
+</phpunit>
```

Comment:
Ok, then we have to add the ability to run single tests to the RoboFile. Having two test runners at the surface is not desirable.

Meanwhile, you can call Codeception directly. http://codeception.com/docs/02-GettingStarted#Running-Tests